main.py

```
@@ -967,6 +1045,39 @@ def _run_e251_job(file_bytes: bytes, source_filename: str):
     return job, profile
 
 
+def _run_e142_job(file_bytes: bytes, source_filename: str):
+    if not vision_service_account_json:
+        raise ValueError("VISION_SERVICE_ACCOUNT_KEY is not configured.")
+
+    job = create_job(kind="e142", source_filename=source_filename)
+    input_pdf_path = job.job_dir / "input.pdf"
+    input_pdf_path.write_bytes(file_bytes)
+    csv_path = job.job_dir / "e142.csv"
+    debug_dir = Path("/tmp") / "plan2table" / "debug" / job.job_id
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)